### DISCUSSION TITLE
The game crashed whilst initializing game Error: java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors, provided by 'fabric-renderer-indigo'! Exit Code: -1

### DISCUSSION THREAD

**Discussion by tankyap** [FabricMC/fabric-api#1719] [Player support]
When I was loaded with Fabric API,it crashes everytime when initializing the game.I decided to put the API only but I still have the same issue.

Fabric loader version: fabric-loader-0.11.6-1.17.1
Fabric API version: fabric-api-0.40.1+1.17

This is a crash report with Fabric API only:
[crash-2021-09-13_20.44.51-client.txt](https://github.com/FabricMC/fabric/files/7154631/crash-2021-09-13_20.44.51-client.txt)

and the exit code: The game crashed whilst initializing game Error: java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors, provided by 'fabric-renderer-indigo'! Exit Code: -1

**Comment by warjort** [comment]
````
Caused by: java.lang.IllegalArgumentException: Malformed \uxxxx encoding.
	at java.base/java.util.Properties.loadConvert(Properties.java:672)
	at java.base/java.util.Properties.load0(Properties.java:455)
	at java.base/java.util.Properties.load(Properties.java:408)
	at net.fabricmc.fabric.impl.client.indigo.Indigo.<clinit>(Indigo.java:105)
````

There is something wrong with your ````config/fabric/indigo-renderer.properties```` configuration file.
I think the error means it contains an invalid character?

You can try deleting it and it should recreate it with default values.
You might want to copy it somewhere else first.

**Reply by tankyap** [reply]
It works,thx

**Reply by JAckwald** [reply]
> ```
> Caused by: java.lang.IllegalArgumentException: Malformed \uxxxx encoding.
> 	at java.base/java.util.Properties.loadConvert(Properties.java:672)
> 	at java.base/java.util.Properties.load0(Properties.java:455)
> 	at java.base/java.util.Properties.load(Properties.java:408)
> 	at net.fabricmc.fabric.impl.client.indigo.Indigo.<clinit>(Indigo.java:105)
> ```
> 
> There is something wrong with your `config/fabric/indigo-renderer.properties` configuration file. I think the error means it contains an invalid character?
> 
> You can try deleting it and it should recreate it with default values. You might want to copy it somewhere else first.

I had the same problem in the version 1.19.3  but it creates the exact same wrong file again. Do you have any idea why thats the case?

**Reply by dawesome35** [reply]
me too :(

**Reply by Pablouche** [reply]
I had the same problem and for me it was the ReplayMod.

**Comment by BboyZeroOne** [comment]
> I had the same problem and for me it was the ReplayMod.
Make sure you remove optifabric file from your mod folder, it kinda clashes with the other mods.

**Reply by lexirosem123** [reply]
I am currently trying to install the ReplayMod and have tried so many different ways, but it will not work. the only mods I have in my folder ar Fabric API and The ReplayMod, but I keep getting this same exit Code -1 
I tried the stuff in the original post replay and I have uninstalled and reinstalled minecraft launcher a few times but nothing seems to work.. any ideas?

**Reply by Poopooracoocoo** [reply]
@lexirosem123 it's impossible to say without a latest.log or crash report. Create a new discussion. You'll find instructions on how to find your latest.log. https://github.com/orgs/FabricMC/discussions/categories/player-support

And for anyone else Googling "fabric error code 1", please know that it doesn't really mean anything. It just means "something went wrong". It's about as useful as the ":(" in a Windows BSOD. Make a new post here!

**Reply by lexirosem123** [reply]
I just started a new discussion and included the crash report. let me know if you need to know more..

Thank you!
https://github.com/orgs/FabricMC/discussions/3337#top

**Comment by Elevenses-11** [comment]
I am having the same apparent problem when attempting to launch Fabric 1.21. I have removed all mods and other config files, but still no success. Could there be something else lurking in the Minecraft directory that  fabric-renderer-indigo doesn't like? 
Any help or suggestions will be greatly appreciated!

**Reply by Poopooracoocoo** [reply]
You should make a new discussion and provide required information such as the log and crash report.

**Comment by steamyguy101** [comment]
how do i find the config/fabric/indigo-renderer.properties file?

**Reply by Poopooracoocoo** [reply]
On Minecraft Launcher for Windows, it'll be in `%appdata%/.minecraft`
